### PR TITLE
Replace '(Mac)OS X' with 'macOS'

### PR DESCRIPTION
**Pull request by miguelarauj1o**
OS X is the old name.
macOS is the official name now.

### CODE REVIEW

**Comment by miguelarauj1o** [comment]
@josh64x2, yes. But, what was I thinking to do that?! Sorry.
I addressed the requested changes.

File: README.md [hunk 1]
```diff
@@ -13,8 +13,8 @@ Additionally or alternatively, you can also sync with a server supporting the [O
 Compatibility
 -------------
 
-Vienna 3.1.x requires a minimum of Mac OS X 10.8 (Mountain Lion).
-The next version of Vienna (3.2.x) will require a minimum of Mac OS X 10.9 (Mavericks).
+Vienna 3.1.x requires a minimum of macOS 10.8 (Mountain Lion).
```
Comment (Eitot) [inline README.md]: Change this into `OS X` instead.

File: README.md [hunk 2]
```diff
@@ -13,8 +13,8 @@ Additionally or alternatively, you can also sync with a server supporting the [O
...
+The next version of Vienna (3.2.x) will require a minimum of macOS 10.9 (Mavericks).
```
Comment (Eitot) [inline README.md]: Change this to `OS X` instead.

File: README.md (hunk 1 above)
Comment (miguelarauj1o) [inline README.md]: ok

File: README.md (hunk 2 above)
Comment (miguelarauj1o) [inline README.md]: ok

**Comment by miguelarauj1o** [comment]
btw, the description project needs to be updated too.

> Vienna is a freeware, open source RSS/Atom newsreader for the Mac OS X operating system.

**Comment by Eitot** [comment]
Also revert changes to the CHANGES file. We shouldn’t change the name retroactively.

**Comment by josh64x2** [comment]
thanks @miguelarauj1o :)

**Pull request merged**